### ISSUE TITLE
Dragon carry

### ISSUE THREAD

**Issue by Aintnobodyreal** [AlexModGuy/Ice-and-Fire-Suggestions#1378]
Dragons are big and strong. It would be an interesting feature if they could pick up entities with their claws and carry (and drop them).
This could be expanded to a set amount of blocks (combined with one of the existing physic mods maybe). If the dragon drops them, they deal damage

**Comment by Saber130** [comment]
Yeah they should make the dragon can carry as much as its feet size
